regression/cbmc-java/stack_var1/stack_var1.j

```diff
@@ -0,0 +1,22 @@
+.class public stack_var1
```

Comment:
Ah, cool. Would you mind adding information in either the commit message or some extra readme file so that these steps become reproducible? I had naively thought that the .class files had been produced from the accompanying .java.